src/topic/TopicInfo.js

```diff
@@ -178,4 +172,23 @@ export default class TopicInfo {
             },
         };
     }
+
+    /**
+     * @param {Uint8Array} bytes
+     * @returns {TopicInfo}
+     */
+    static fromBytes(bytes) {
+        const temp = {
+            topicInfo: proto.ConsensusTopicInfo.decode(bytes)
+        }
+        return TopicInfo._fromProtobuf(/** @type {proto.IConsensusGetTopicInfoResponse} */ (temp));
```

Comment:
Why don't you just do instead of `temp`
```javascript
return TopicInfo._fromProtogbuf({ topicInfo: proto.ConsensusTopicInfo.decode(bytes) });`
```